packages/eslint-plugin-js/rules/comma-style/comma-style.ts

```diff
@@ -3,14 +3,18 @@
  * @author Vignesh Anand aka vegetableman
  */
 
+import type { TSESTree } from '@typescript-eslint/types'
+import type { RuleFunction } from '@typescript-eslint/utils/ts-eslint'
```

Comment:
I'd like to avoid directly reference to `@typescript-eslint` as we don't deps on it. Please import `Tree` and `RuleFunction` from `'../../utils/types'`